src/libos/src/process/do_wait4.rs

```diff
@@ -54,7 +58,8 @@ pub fn do_wait4(child_filter: &ProcessFilter, options: WaitOptions) -> Result<(p
         .collect::<Vec<&ProcessRef>>();
 
     if unwaited_children.len() == 0 {
-        return_errno!(ECHILD, "Cannot find any unwaited children");
+        // Check again for vfork-ed child process which doesn't have a real structure of a process
+        return wait4_exit_child_created_with_vfork(process.pid(), child_filter);
```

Comment:
If there are child processes and all child processes are running, the vfork-ed child process of the terminated should also be checked.

Reply:
Fixed. Thanks for your review.